src/components/Todo/EditCard/edit-card.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect, useRef } from "react";
+import PropTypes from "prop-types";
+import EditActionBar from "components/Todo/EditCard/edit-action-bar.component";
+import { ENTER } from "common/constants";
+
+const EditCard = ({
+  todo,
+  handleEditTask,
+  handleCompleteTask,
+  handleEditCancelTask,
+}) => {
+  const { id, task } = todo;
+  const textRef = useRef(null);
+  const [editedTask, setEditData] = useState(task);
```

Comment:
```suggestion
  const [editedTask, updateTask] = useState(task);
```

Reply:
```suggestion
  const [editedTask, setEditedTask] = useState(task);
```

Is this okay to use that?